src/com/github/argajuvi/Main.java

```diff
@@ -1,49 +1,51 @@
 package com.github.argajuvi;
 
-import com.github.argajuvi.models.order.Order;
+import com.github.argajuvi.menus.AdminMenu;
+import com.github.argajuvi.menus.Menu;
+import com.github.argajuvi.menus.UserMenu;
 import com.github.argajuvi.models.product.BookProduct;
 import com.github.argajuvi.models.product.ClothingProduct;
 import com.github.argajuvi.models.product.FoodProduct;
 import com.github.argajuvi.models.product.Product;
-import com.github.argajuvi.models.receipt.Receipt;
 import com.github.argajuvi.models.user.User;
 import com.github.argajuvi.utils.Utils;
-import com.sun.istack.internal.Nullable;
 
 import java.time.LocalDate;
-import java.time.format.DateTimeParseException;
 import java.time.temporal.ChronoUnit;
-import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class Main {
 
-    private final List<Product> productList;
-    private final List<User> userList;
-
-    private User currentUser = null;
+    public static List<Product> productList = Stream.of(

```

Comment:
I just remembered, semua yg `public static` bagusnya dikasih nama upper snake-case, misalnya ini `productList` jadinya `PRODUCT_LIST`. Do you want me to take care of this or nah?

Reply:
lemme do it. gw baru tau ttg ini, kirain cuma konstanta

Reply:
I mean... technically itu harusnya konstanta, `List` tuh kalo di `final` masih tetap mutable.